x/perpetual/spec/09_funding_payment.md

```diff
@@ -42,8 +42,8 @@ This formula ensures that the funding rate stays within defined boundaries, ince
 
 ### 4. **Funding Fee Collection**
 
-At every block, the calculated funding fees are collected and sent to the **Funding Fee Collection Address**. The details of this process can be found in the source code:  
-[Funding Fee Collection Code](https://github.com/elys-network/elys/blob/main/x/perpetual/keeper/handle_funding_fee_collection.go).
+At every block, the calculated funding fees are tracked. If a user makes a interaction(close position or consolidate) the fee collection is triggered and The details of this process can be found in the source code:  
```

Comment:
@amityadav0 "fee collection is triggered and…" the sentence seems to missing some words